docs/en/sql-reference/aggregate-functions/reference/quantiles.md

```diff
@@ -114,3 +114,47 @@ Result:
 │ [249.75,499.5,749.25,899.1,949.05,989.01,998.001]                   │
 └─────────────────────────────────────────────────────────────────────┘
 ```
+
+## quantilesGK
```

Comment:
I'd suggest to mentioning in the function name the fact that it is approximate, not the concrete algorithm.


```suggestion
## quantilesApprox
```

Reply:
Could you please also provide some examples to hightling the difference between our current implementation, where it can be beneficial (maybe including time and memory usage).

Trivial example doesn't show the strong sides, but I suppose they are:


```
 :) select quantile( 0.5)(number) from numbers_mt(1_000_000_000);

┌─quantile(0.5)(number)─┐
│             505912435 │
└───────────────────────┘

1 row in set. Elapsed: 0.873 sec. Processed 1.00 billion rows, 8.00 GB (1.15 billion rows/s., 9.17 GB/s.)

 :) select quantileGK(1000, 0.5)(number) from numbers_mt(1_000_000_000);

┌─quantileGK(1000, 0.5)(number)─┐
│                     499804247 │
└───────────────────────────────┘

1 row in set. Elapsed: 6.413 sec. Processed 1.00 billion rows, 8.00 GB (155.93 million rows/s., 1.25 GB/s.)

 :) select quantileGK(10000, 0.5)(number) from numbers_mt(1_000_000_000);

┌─quantileGK(10000, 0.5)(number)─┐
│                      499710915 │
└────────────────────────────────┘

1 row in set. Elapsed: 11.343 sec. Processed 1.00 billion rows, 8.00 GB (88.16 million rows/s., 705.27 MB/s.)
```